glean-core/python/setup.py

```diff
@@ -126,6 +118,34 @@ def finalize_options(self):
             self.install_lib = self.install_platlib
 
 
+class build(_build):
+    def run(self):
+        try:
+            subprocess.run(["cargo"])
+        except subprocess.CalledProcessError:
+            print("glean_sdk requires 'cargo' to build its Rust extension.")
+            sys.exit(1)
+
+        command = ["cargo", "build", "--all"]
```

Comment:
Probably better to build only glean-ffi (`--all` probably also buils glean-preview, which we don't need)

```suggestion
        command = ["cargo", "build", "--package", "glean-ffi"]
```